mp4parse_capi/src/lib.rs

```diff
@@ -248,7 +248,7 @@ impl Read for mp4parse_io {
 
 /// Allocate an `mp4parse_parser*` to read from the supplied `mp4parse_io`.
 #[no_mangle]
-pub unsafe extern fn mp4parse_new(io: *const mp4parse_io) -> *mut mp4parse_parser {
+pub unsafe extern fn mp4parse_new(io: *const mp4parse_io, log: bool) -> *mut mp4parse_parser {
```

Comment:
Right now I enable rust log when prlog is on for MP4Metadta. I'll think how to forward rust log into gecko's log buffer.